From c0f6e8fe4c3b6803be97aeea6683631d616412f4 Mon Sep 17 00:00:00 2001
From: Alex Buynytskyy <alexbuy@google.com>
Date: Thu, 08 Dec 2022 17:52:52 +0000
Subject: [PATCH] Revert "Hacky workaround for half-finalized builds."

This reverts commit 12eea1512f2612f41b5cf7004ee2e6a189d548d7.

Reason for revert: finalization-2

Change-Id: Ifc801271628808693b1cb20206f8f81c9a6c694d
---

diff --git a/java/droidstubs.go b/java/droidstubs.go
index ec4a0f4..5777b18 100644
--- a/java/droidstubs.go
+++ b/java/droidstubs.go
@@ -386,8 +386,7 @@
 	}
 	if apiVersions != nil {
 		cmd.FlagWithArg("--current-version ", ctx.Config().PlatformSdkVersion().String())
-		// STOPSHIP: RESTORE THIS LOGIC WHEN DECLARING "REL" BUILD
-		// cmd.FlagWithArg("--current-codename ", ctx.Config().PlatformSdkCodename())
+		cmd.FlagWithArg("--current-codename ", ctx.Config().PlatformSdkCodename())
 		cmd.FlagWithInput("--apply-api-levels ", apiVersions)
 	}
 }
